WordPress/Classes/Models/Media.m

```diff
@@ -261,4 +261,20 @@ - (BOOL)hasRemote {
     return self.mediaID.intValue != 0;
 }
 
+- (void)setError:(NSError *)value
```

Comment:
**Nit:**
```
- (void)setError:(NSError *)error
```

Reply:
Good catch! The setter was created automatically via Xcode and I should've cleaned that up. Done.